solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -1050,34 +1029,37 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
         uint256 _reserveTokenIndex,
         uint256 _reserveAmount
     ) public view returns (uint256[2] memory) {
-        uint256[2] memory _reserveAmounts;
+        uint256[2] memory reserveAmounts;
 
         uint256 totalSupply = IDSToken(address(anchor)).totalSupply();
-        uint256[2] memory _reserveBalances = baseReserveBalances(_reserveTokens);
-        uint256 amount = fundSupplyAmount(totalSupply, _reserveBalances[_reserveTokenIndex], _reserveAmount);
+        uint256[2] memory balancesMinusFees = baseReserveBalances(_reserveTokens);
```

Comment:
balancesMinusFees -> baseBalances (keep it consistent)

Reply:
done